hardware_applications/week1/starter_code.cpp

```diff
@@ -0,0 +1,5 @@
+#include <iostream>
+#include <STSL/RJRobot.h>
+
+using namespace std;
+
```

Comment:
Since the hard part of this week is getting anything moving, it would be nice to have the most basic code in the starter, IE: spin one motor, just to know that it's working well before people start writing code.

Reply:
These hardware demos are designed to exercise concepts we teach each week. So, this week, we want them to go through writing the main signature and just basic C++ setup. 
minimal "spin a motor" code is really straightforward and would be a good first step for guiding people through the demo:
```
int main() {
    RJRobot robot;
    robot.SetMotor(MotorPort::A, 200);
    return 0;
}
```